nixos/cwm: init. Added windowManager cwm, basing on the module for dwm.

mkf opened the pull request:
Added windowManager cwm, basing on the module for dwm.

<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#sec-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
###### Motivation for this change

Necessary to enable cwm window manager as a choice from display managers.

###### Things done

Added `config.services.xserver.windowManager.cwm` just like `config.services.xserver.windowManager.dwm`.

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @0x4A6F

mkf committed:
nixos/cwm: init

Added windowManager cwm, basing on the module for dwm.

romildo reviewed (changes requested):
Please, add `cwm` to the list of imports in https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/window-managers/default.nix#L14

mkf committed:
nixos/window-managers: add cwm to imports

as per request from @romildo under #70035 https://github.com/NixOS/nixpkgs/pull/70035#pullrequestreview-298166798

romildo reviewed (approved):
Tested on a VM.
